R/navs-legacy.R

```diff
@@ -347,17 +415,23 @@ navs_bar_ <- function(..., title = NULL, id = NULL, selected = NULL,
                       theme = NULL) {
 
   if (identical(inverse, "auto")) {
-    inverse <- TRUE
-    if (identical(theme_preset_info(theme)$name, "shiny")) {
-      inverse <- FALSE
+    if (is.null(theme) || theme_version(theme) < 5) {
+      inverse <- TRUE
```

Comment:
I think I'm OK with this, but would you mind writing down the thought process behind this? 

Also, this will change navbar appearance for Bootswatch themes (and vanilla Bootstrap), so assuming we're OK with that, let's make sure to call it out as a breaking change in the NEWS

Reply:
Sorry, I think the code is easier to review with the smaller PRs, but the context is a bit spread out. The motivation is outlined in the first part of the description in https://github.com/rstudio/bslib/pull/1145.

In short, in Bootstrap 5 `inverse = type = "auto"` now means "match the current light/dark mode". Before this, it `"auto"` meant "pick for me". 

We _could_ get into setting `class` and `type` attributes here to get navbar colors that match the current defaults. The continuity would be nice, but it'd add work and complexity that I'm not sure is worth the effort.